packages/nodes-base/nodes/Github/GithubTrigger.node.ts

```diff
@@ -31,10 +31,14 @@ export class GithubTrigger implements INodeType {
 		inputs: [],
 		outputs: ['main'],
 		credentials: [
+			// {
+			// 	name: 'githubApi',
+			// 	required: true,
+			// }, 
```

Comment:
We can see the history of the changes online `35`, so there's no need to keep old code around. It's safe to remove these comments.

Reply:
Sounds good! I just made the suggested changes.